packages/react-compose/src/resolveSlotProps.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { OptionsResolverResult, defaultMappedProps } from './createOptionsResolver';
+
+export const NullRender = () => null;
+
+/**
+ * Helper utility which takes in a classes array from compose options, resolves functions,
+ * merges them into a final result, and distributes classnames to slotProps within the given
+ * resolver result object.
+ */
+export function resolveSlotProps(result: OptionsResolverResult): OptionsResolverResult {
+  const { state, slots, slotProps } = result;
+
+  Object.keys(slots).forEach((slotName: string) => {
+    const slot = slots[slotName];
+    let slotProp = state[slotName];
+
+    if (slot && slotProp !== undefined && slotProp !== null) {
```

Comment:
Hey, why do you check slot to see if it's falsey but check slotProp and make sure it's neither undefined or null? Is something like `false` a valid slot prop?

Reply:
slot: null is valid, and should skip the logic. slotProp={null} I wasn't sure how to handle so I assumed it to mean don't render this slot.

```
<Button chevron={null} />
```